MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1338,6 +1343,11 @@
 	addMTLVertexFormatDesc( Half, None, None );
 
 	addMTLVertexFormatDesc( UChar4Normalized_BGRA, None, None );
+	
+#if MVK_XCODE_15
+	addMTLVertexFormatDesc( FloatRG11B10, Vertex, Vertex );
+	addMTLVertexFormatDesc( FloatRGB9E5, Vertex, Vertex );
```

Comment:
These should be `None`, because these formats weren't added until macOS 14/iOS 17.